### PR TITLE
Fixed get method

### PR DESCRIPTION
**Pull request by macieg**
Unfortunately fxn values are not arrays.

### CODE REVIEW

**Comment by iandees** [comment]
Can we rename the function something like "get_from_array_string"?

**Comment by macieg** [comment]
@iandees done

File: openaddr/conform.py [hunk 1]
```diff
@@ -967,7 +967,7 @@ def row_function(sd, row, key, fxn):
     elif function == "first_non_empty":
         row = row_fxn_first_non_empty(sd, row, key, fxn)
     elif function == "get":
```
Comment (iandees) [inline openaddr/conform.py:969]: I meant the name of the function in the JSON here.

I think we might also need to add the function name to [the JSON schema](https://github.com/openaddresses/openaddresses/blob/master/schema/source_schema.json) in the openaddresses repo.

Comment (macieg) [reply openaddr/conform.py]: Done.

I've already changed the schema in my draft PR https://github.com/openaddresses/openaddresses/pull/4987/commits/bd63f8ab00d13deee1fac34c0db4f2d06744f3cd

**Pull request merged**